src/Faithlife.Data/DbConnectorCommand.cs

```diff
@@ -30,6 +30,11 @@ namespace Faithlife.Data
 		/// </summary>
 		public DbConnector Connector { get; }
 
+		/// <summary>
+		/// True after <seealso cref="Prepare"/> is called.
```

Comment:
```suggestion
		/// True after <see cref="Prepare"/> is called.
```

Was `seealso` intentional?

Reply:
No.